src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,20 @@
 // Vendor-specific styles.
+
+//UIO
+.flc-toc-tocContainer {
+	background-color: $white;
+	padding-top: rem(39.84);
+
+	.flc-toc-header {
+		margin-top: 0;
+	}
+}
+
+.flc-toc-tocContainer * {
+	margin: $white;
+}
+
+.fl-prefsEditor-buttons {
```

Comment:
adding styling to the span does not work for me as expected

Reply:
It's because the specificity you defined is lower than what comes with UIO. UIO uses `.fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons` to style this container. Using the same or higher specificity selector should override the UIO default.

Reply:
This was still not working after using higher specificity so I implemented `!important` as a temporary fix

Reply:
I issued a pull request https://github.com/TeddTech/wecount.inclusivedesign.ca/pull/3 against your branch to demonstrate using a higher specificity css selector does solve the problem. Please have a look.